### PR TITLE
Update information for developer

### PR DESCRIPTION
**Pull request by doncem**
### JIRA link (if applicable) ###

[Create send-letter onboarding page in Confluence](https://tools.hmcts.net/jira/browse/BPS-1049)

### Change description ###

In order to start populating Confluence page I need to make sure dockerised application boot works. Including sample docker compose file and extending it with sftp and auth provider services.

Updated onboarding section as it is very minimal for Send Letter Service.

Note. There is a dummy sftp public key (included with separate commit but necessary for docker boot). Will have to look into separately to solve configuration of sftp service connectivity. ATM I am receiving `Invalid MD5 fingerprint: fingerprint` which is obvious error. Service on it's own is operating though. Need to upgrade documentation first.

tl;dr next PR - solve sftp config

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,14 @@
 
 [Swagger API Specs](https://hmcts.github.io/reform-api-docs/swagger.html?url=https://hmcts.github.io/reform-api-docs/specs/send-letter-service.json)
 
+## Table of Contents
+
+* [Running the application](#running-the-application)
+    * [Locally](#locally)
+    * [Docker environment](#docker-environment)
+* [Onboarding new services](#onboarding-new-services)
+* [License](#license)
```
Comment (rkondratowicz) [inline README.md]: nice

File: README.md [hunk 2]
```diff
@@ -21,47 +29,64 @@ $ ./gradlew bootRun
...
+
+```yaml
+ftp:
+  service-folders:
+  - service: send_letter_tests
+    folder: BULKPRINT
+```
+
+**Note!**
+This can only be deployed once ftp client has created folder in all relevant environments.
```
Comment (rkondratowicz) [inline README.md]: 'ftp client'?

Comment (doncem) [reply README.md]: perhaps provider :)

File: README.md [hunk 3]
```diff
@@ -21,47 +29,64 @@ $ ./gradlew bootRun
...
 ```bash
-$ docker ps -a
-```
+curl -X POST -H "Content-Type: application/json" "http://localhost:4552/lease" -d '{"microservice":"send_letter_tests","oneTimePassword":"OTP"}'
 
-CONTAINER ID | IMAGE | COMMAND | CREATED | STATUS | PORTS | NAMES
------------- | ----- | ------- | ------- | ------ | ----- | -----
-9e3fc00c1824 | hmcts/send-letter-service | "/usr/bin/java -jar …" | 4 minutes ago | Up 4 minutes (healthy) | 0.0.0.0:8485->8485/tcp | bsp-send-letter-service
-e68f86e63c93 | send-letter-database | "docker-entrypoint.s…" | 36 minutes ago | Up 36 minutes (healthy) | 5432/tcp | bsp-send-letter-database
+OTP_RESPONSE
```
Comment (nitinprabhu) [inline README.md]: this is token ? otp response will be a number ?

File: README.md [hunk 4]
```diff
@@ -21,47 +29,64 @@ $ ./gradlew bootRun
...
 
-Test:
+curl -X POST -H "Content-Type: application/vnd.uk.gov.hmcts.letter-service.in.letter.v2+json" -H "ServiceAuthorization: Bearer OTP_RESPONSE" "http://localhost:8485/letters" -d '{"documents":["aGVsbG8="],"type":"BPS001"}'
```
Comment (nitinprabhu) [inline README.md]: OTP_RESPONSE -> S2S token

File: README.md (hunk 3 above)
Comment (doncem) [reply README.md]: OTP is OTP. 6 digit string from google auth. response is s2s token. i can rename latter one ;)
